src/Data/Uniques/bow.lua

```diff
@@ -73,48 +73,74 @@ Variant: Minion Attack and Cast Speed (Pre 3.17.0)
 Variant: Double Damage (Pre 3.17.0)
 Variant: Double Damage while Focused (Pre 3.17.0)
 Variant: Socketed Spell Trigger (Pre 3.17.0)
-Variant: Pre 3.17.0
+Variant: Pre 3.17.0 [14]
```

Comment:
I believe this is a typo
```suggestion
Variant: Pre 3.17.0
```